Quaver.Shared/Converters/Osu/Osu.cs

```diff
@@ -70,7 +70,7 @@ public static void ConvertOsz(string file, string extractDirectory)
                     var fileName = $"{extractDirectory}/{Path.GetFileName(tempFile)}";
 
                     // Make sure the path to the file is less than 200 characters
```

Comment:
This comment should be either updated or removed.

Reply:
Comment fixed